From 2cddd1c79f51a37f3fa82e3c58518cf097a75a55 Mon Sep 17 00:00:00 2001
From: Matthias Schoepfer <m.schoepfer@rethinkrobotics.com>
Date: Tue, 9 Apr 2024 14:04:26 +0200
Subject: [PATCH] Revert install of .so files into python path

There seems that some regression might have happened after #195.
When removing those 2 lines, we avoid to install the .so files
in lib *and* python path.

Upstream-Status: Submitted [https://github.com/ros2/rosidl_python/pull/211]

Signed-off-by: Matthias Schoepfer <m.schoepfer@rethinkrobotics.com>
---
 .../cmake/rosidl_generator_py_generate_interfaces.cmake       | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake b/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake
index ba49190..cfc424a 100644
--- a/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake
+++ b/rosidl_generator_py/cmake/rosidl_generator_py_generate_interfaces.cmake
@@ -236,10 +236,6 @@ endforeach()
 
 set_target_properties(${_target_name_lib} PROPERTIES COMPILE_OPTIONS "${_extension_compile_flags}")
 
-set_target_properties(${_target_name_lib} PROPERTIES
-  LIBRARY_OUTPUT_DIRECTORY ${_output_path}
-  RUNTIME_OUTPUT_DIRECTORY ${_output_path})
-
 if(NOT rosidl_generate_interfaces_SKIP_INSTALL)
   install(TARGETS ${_target_name_lib}
     EXPORT export_${_target_name_lib}
-- 
2.34.1

